lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1116,7 +1115,9 @@ sub update_status {
         return $ret;
     }
 
-    $self->append_log($status->{log});
+    $self->append_log($status->{log},             "autoinst-log-live.txt");
+    $self->append_log($status->{serial_log},      "serial-terminal-live.txt");
+    $self->append_log($status->{serial_terminal}, "serial-terminal-live.txt");
```

Comment:
I don't understand, you append both serial_log and serial_terminal to `serial-terminal-live.txt`?